compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -29,7 +29,7 @@ impl MutVisitor for Marker {
 enum Frame<'a> {
     Delimited {
         tts: &'a [mbe::TokenTree],
-        delim_token: token::DelimToken,
+        delim_token: token::Delimiter,
```

Comment:
One more thing: there are quite a few `delim_token` identifiers that could be renamed to `delim` or `delimeter`, which you can find with grep.